src/Libraries/DSOffice/Excel.cs

```diff
@@ -321,14 +321,15 @@ public static object[][] Read(string filePath, string sheetName)
         /// </param>
         /// <param name="data">Data to write to the spreadsheet.</param>
         /// <param name="overWrite"></param>
+        /// <param name="writeAsString">Toggle to switch between writing Excel file as strings.</param>
         /// <returns name="data">Data written to the spreadsheet.</returns>
         /// <search>office,excel,spreadsheet</search>
         [IsVisibleInDynamoLibrary(false)]
-        public static object[][] WriteToFile(string filePath, string sheetName, int startRow, int startCol, object[][] data, bool overWrite = false)
+        public static object[][] WriteToFile(string filePath, string sheetName, int startRow, int startCol, object[][] data, bool overWrite = false, bool writeAsString = false)
```

Comment:
This function is having a binary breaking change because optional parameters are a compile-time feature. Can we use an overload instead?

Reply:
Sorry, hadnt realized this was public too, will fix that.

Reply:
This new method doesn't need to be public. This logic can actually be folded into the new `ExportToExcel` node. The only reason there existed an old `WriteToFile` method was that it was originally a node, which was later renamed to `ExportExcel`.

Reply:
So one thing before i make this change. Would you prefer to have one private method that `WriteToFile` and `ExportToExcel` uses? other option would be to duplicate the logic from `WriteToFile` into `ExportToExcel` with the additional parameter.

Reply:
I think option 1 sounds good. I'd rather not duplicate the code if possible.